### ISSUE TITLE
paypalrestsdk==2.0.0b1 is on pypi but not installable

### ISSUE THREAD

**Issue by jleclanche** [paypal/PayPal-Python-SDK#214]
```
% pip install paypalrestsdk==2.0.0b1
Collecting paypalrestsdk==2.0.0b1
  Downloading paypalrestsdk-2.0.0b1-py2.py3-none-any.whl (43kB)
    100% |████████████████████████████████| 51kB 331kB/s 
Collecting braintreehttp>=1.2.3 (from paypalrestsdk==2.0.0b1)
  Could not find a version that satisfies the requirement braintreehttp>=1.2.3 (from paypalrestsdk==2.0.0b1) (from versions: 0.1.0, 0.1.1, 0.1.3, 0.2.0)
No matching distribution found for braintreehttp>=1.2.3 (from paypalrestsdk==2.0.0b1)
```

Any idea what's up with this? Is the 2.0.0b1 release a mistake?

**Comment by pp-randy** [comment]
Its a beta release.  I will follow up on the braintreehttp library not being available.

**Comment by sdcoffey** [comment]
Hey @jleclanche,

Sorry about that! There are still some moving pieces in play while the 2.0 version of this library, and the supporting `braintreehttp` library are under development. I've fixed this issue in `2.0.0b2`. Please let us know if you experience any other issues, or have any other feedback! Thanks!

**Comment by jleclanche** [comment]
Thanks!
There's not a lot of documentation about 2.0.0 and the changes. Is it just a prototype or is it definitely what you guys are moving towards? I'm building a new django integration and it'd be nice to know if i should be using the 2.0 one :)
